bindings/cpp/Makefile

```diff
@@ -30,7 +30,7 @@ ifeq "$(THREADING)" "pthread"
 override CPPFLAGS += -DGTEST_HAS_PTHREAD=1 -DWTF_PTHREAD_THREADED
 override LDLIBS += -pthread
 else ifeq "$(THREADING)" "std"
-override CPPFLAGS += -DGTEST_HAS_PTHREAD=1
+override CPPFLAGS += -DGTEST_HAS_PTHREAD=1 -DWTF_STD_THREADED
```

Comment:
Sorry for taking so long to respond to the initial bug report. This project is one of many part-time things for me.

We were hoping that the default, standards compliant behavior, could work without any additional macro definitions, and then we could only use an additional threading define for platforms that are broken or purposefully single threaded (we select the latter for one bare metal target that runs without a real OS). Currently, I think the only generally available "broken" platform is Android (although we do use this on a couple of other variants that need pthreads vs std threads for one reason or another).

This isn't just theoretical -- not adding a required threading macro makes this work out of the box when used in a Windows project.

Wdyt of just changing the makefile to default to THREADING="std"? The only reason that pthread was in here as the default is that it predated the std implementation due to this library being first developed and used on a platform that only supported pthread. We should then update the comments and/or README to note that when building for android, you should specify pthread threading by way of the THREADING=pthread to the Makefile or defining -DWTF_PTHREAD_THREADED.